recipe/meta.yaml

```diff
@@ -19,21 +19,21 @@ source:
     - h5py_win_setup_build.py.patch
 
 build:
-  number: 2
+  number: 3
 
 requirements:
   build:
     - python
     - numpy x.x
-    - hdf5
+    - hdf5 1.8*
     - cython
     - six
     - pkgconfig
 
   run:
     - python
     - numpy x.x
-    - hdf5
+    - hdf5 1.8*
```

Comment:
I guess that in this case conda will choose `hdf5 1.8.16` instead of `hdf5 1.8.15.1` when building, correct?

But will allow for `hdf5 1.8.15.1` to be installed later on breaking the package, no?